src/helper/passwordHelpers.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+	checkPasswordStrength:
+		password => {
+			const passwordPattern = new RegExp('^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z])(?=.*[\\-_!<>§$%&\\/()=?\\\\;:,.#+*~\']).{8,255}$');
```

Comment:
Codacy found an issue: [This line has a length of 134. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)